tests/integration/security/util/framework.go

```diff
@@ -42,14 +43,17 @@ const (
 )
 
 type EchoDeployments struct {
-	Namespace1       namespace.Instance
-	A, B, C, D, E, F echo.Instances
-	Multiversion     echo.Instances
-	Headless         echo.Instances
-	Naked            echo.Instances
-	VM               echo.Instances
-	HeadlessNaked    echo.Instances
-	All              echo.Instances
+	// Namespace1 is used as the default namespace for reachability tests and other tests which share the same config
+	Namespace1 namespace.Instance
+	// Namespace2 is used for echos used by TestPassThroughFilterChain
```

Comment:
why does this test needs it own namespace and 4 more echos? can it use the same pods?

Reply:
Per comment above, if we can do config switch, we do not need more echos.